### PR TITLE
[Behat] Changed sample data to correspond with default configuration

### LINKED ISSUE #5707: [Behat] Adjust sample channels and currency
Legacy features are gone, so let's replace France with United States and euros with dollars. ;) To make it consistent with default configuration. :D (no other reasons, sorry our French friends!)

### PR DESCRIPTION
**Pull request by NeverResponse**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | yes
| Related tickets | fixes #5707
| License         | MIT

- [x] Moved sample store to United States
- [x] Exchanged Euros with Dollars
- [x] Scenarios logic clean-ups

### CODE REVIEW

File: features/addressing/managing_zones/zone_validation.feature [hunk 1]
```diff
@@ -40,10 +40,10 @@ Feature: Zone validation
     Scenario: Seeing a disabled type field when adding country type zone
         Given I want to create a new zone consisting of country
         Then the type field should be disabled
-        And it should be country type
+        And it should be of country type
```
Comment (michalmarcinkowski) [inline features/addressing/managing_zones/zone_validation.feature]: Are you sure about this change?

File: features/checkout/shipping_order/preventing_not_available_shipping_method_selection.feature [hunk 2]
```diff
@@ -5,51 +5,52 @@ Feature: Preventing not available shipping method selection
...
         Then I should not be able to select "Raven Post" shipping method
 
     @ui
     Scenario: Being alerted about no shipping method available
-        Given there is a zone "EU" containing all members of the European Union
+        Given there is zone "NA" containing all members of the North America
         And there is a zone "The Rest of the World" containing all other countries
-        And the store has "Dragon Post" shipping method with "€30.00" fee for the rest of the world
-        And the store has disabled "Raven Post" shipping method with "€10.00" fee
+        And the store has "Dragon Post" shipping method with "$30.00" fee within "RoW" zone
```
Comment (michalmarcinkowski) [inline features/checkout/shipping_order/preventing_not_available_shipping_method_selection.feature]: What was wrong with `for the rest of the world`?

File: features/checkout/shipping_order/preventing_not_available_shipping_method_selection.feature [hunk 3]
```diff
@@ -5,51 +5,52 @@ Feature: Preventing not available shipping method selection
...
         When I am at the checkout addressing step
-        And I specify the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
+        And I specify the shipping address as "Ankh Morpork", "Frost Alley", "90210", "United States" for "Jon Snow"
         And I complete the addressing step
         Then I should not be able to select "Dragon Post" shipping method
 
     @ui
     Scenario: Not being able to select shipping method not available for my shipping address
-        Given there is a zone "EU" containing all members of the European Union
+        Given there is zone "NA" containing all members of the North America
```
Comment (michalmarcinkowski) [inline features/checkout/shipping_order/preventing_not_available_shipping_method_selection.feature]: there is **a** zone

File: features/checkout/shipping_order/preventing_not_available_shipping_method_selection.feature [hunk 4]
```diff
@@ -5,51 +5,52 @@ Feature: Preventing not available shipping method selection
...
         And there is a zone "The Rest of the World" containing all other countries
-        And the store has "Dragon Post" shipping method with "€30.00" fee within "EU" zone
-        And the store has "Raven Post" shipping method with "€10.00" fee for the rest of the world
+        And the store ships to "Germany"
+        And the store has "Dragon Post" shipping method with "$30.00" fee for the rest of the world
+        And the store has "Raven Post" shipping method with "$10.00" fee within "NA" zone
```
Comment (michalmarcinkowski) [inline features/checkout/shipping_order/preventing_not_available_shipping_method_selection.feature]: If you wouldn't change the zones for shipping methods we could use `United States` as shipping address and get rid of irrelevant step `store ships to Germany`

File: features/order/managing_orders/seeing_order_item_included_taxes.feature [hunk 5]
```diff
@@ -5,24 +5,24 @@ Feature: Seeing included in price taxes of order items
...
-        And the store has included in price "Guns tax" tax rate of 10% for "Guns" within "EU" zone
-        And the store has a product "Winchester M1866" priced at "€220.00"
+        Given the store operates on a single channel in "United States"
+        And there is zone "NA" containing all members of the North America
+        And default tax zone is "NA"
+        And the store has included in price "Guns tax" tax rate of 10% for "Guns" within "NA" zone
+        And the store has a product "Winchester M1866" priced at "$220.00"
         And it belongs to "Guns" tax category
-        And the store ships everything for free within "EU" zone
+        And the store ships everything for free within "NA" zone
```
Comment (michalmarcinkowski) [inline features/order/managing_orders/seeing_order_item_included_taxes.feature]: Do we need `NA` zone at all? As default there is a `US` zone created so we can use it for this example.

File: features/order/managing_orders/seeing_order_item_included_taxes.feature [hunk 6]
```diff
@@ -5,24 +5,24 @@ Feature: Seeing included in price taxes of order items
...
 
     Background:
-        Given the store operates on a single channel in "France"
-        And there is a zone "EU" containing all members of the European Union
-        And default tax zone is "EU"
-        And the store has included in price "Guns tax" tax rate of 10% for "Guns" within "EU" zone
-        And the store has a product "Winchester M1866" priced at "€220.00"
+        Given the store operates on a single channel in "United States"
+        And there is zone "NA" containing all members of the North America
+        And default tax zone is "NA"
```
Comment (michalmarcinkowski) [inline features/order/managing_orders/seeing_order_item_included_taxes.feature]: This step is irrelevant for current scenario

File: features/order/managing_orders/seeing_order_item_taxes.feature [hunk 7]
```diff
@@ -5,23 +5,23 @@ Feature: Seeing taxes of order items
...
-        And the store has a product "Symfony2 T-Shirt" priced at "€140.00"
+        Given the store operates on a single channel in "United States"
+        And there is zone "NA" containing all members of the North America
+        And default tax zone is "NA"
+        And the store has "VAT" tax rate of 23% for "Clothes" within "NA" zone
+        And the store has a product "PHP T-Shirt" priced at "$80.00"
+        And the store has a product "Symfony2 T-Shirt" priced at "$140.00"
         And it belongs to "Clothes" tax category
-        And the store ships everything for free within "EU" zone
+        And the store ships everything for free within "NA" zone
```
Comment (michalmarcinkowski) [inline features/order/managing_orders/seeing_order_item_taxes.feature]: Same as above.

File: features/order/managing_orders/seeing_order_shipping_taxes.feature [hunk 8]
```diff
@@ -5,14 +5,14 @@ Feature: Seeing taxes of an order
...
-        And the store has a product "Symfony2 T-Shirt" priced at "€140.00"
+        Given the store operates on a single channel in "United States"
+        And there is zone "NA" containing all members of the North America
+        And default tax zone is "NA"
+        And the store has "VAT" tax rate of 23% for "Clothes" within "NA" zone
+        And the store has "Shipping VAT" tax rate of 23% for "Shipping Services" within "NA" zone
+        And the store has a product "Symfony2 T-Shirt" priced at "$140.00"
         And it belongs to "Clothes" tax category
-        And the store has "DHL" shipping method with "€10.00" fee within "EU" zone
+        And the store has "DHL" shipping method with "$10.00" fee within "NA" zone
```
Comment (michalmarcinkowski) [inline features/order/managing_orders/seeing_order_shipping_taxes.feature]: Same as above.

File: features/order/managing_orders/seeing_order_shipping_total.feature [hunk 9]
```diff
@@ -5,14 +5,14 @@ Feature: Seeing shipping total of an order
...
-        And the store has a product "Gryffindor scarf" priced at "€100.00"
+        Given the store operates on a single channel in "United States"
+        And there is zone "NA" containing all members of the North America
+        And default tax zone is "NA"
+        And the store has "VAT" tax rate of 23% for "Hogwart stuff" within "NA" zone
+        And the store has "Shipping VAT" tax rate of 23% for "Shipping Services" within "NA" zone
+        And the store has a product "Gryffindor scarf" priced at "$100.00"
         And it belongs to "Hogwart stuff" tax category
-        And the store has "Owl post" shipping method with "€10.00" fee within "EU" zone
+        And the store has "Owl post" shipping method with "$10.00" fee within "NA" zone
```
Comment (michalmarcinkowski) [inline features/order/managing_orders/seeing_order_shipping_total.feature]: Same as above.

File: features/order/managing_orders/seeing_orders_payment_amount_including_all_fees.feature [hunk 10]
```diff
@@ -5,14 +5,15 @@ Feature: Seeing order's payment amount including all fees
...
-        Given the store operates on a single channel in "France"
-        And there is a zone "EU" containing all members of the European Union
-        And default tax zone is "EU"
-        And the store has "EU VAT" tax rate of 23% for "Clothes" within "EU" zone
-        And the store has a product "Angel T-Shirt" priced at "€39.00"
+        Given the store operates on a single channel in "United States"
+        And there is zone "NA" containing all members of the North America
+        And there is a zone "The Rest of the World" containing all other countries
+        And default tax zone is "NA"
+        And the store has "VAT" tax rate of 23% for "Clothes" within "NA" zone
```
Comment (michalmarcinkowski) [inline features/order/managing_orders/seeing_orders_payment_amount_including_all_fees.feature]: No need for `NA` zone

File: features/order/managing_orders/seeing_orders_payment_amount_including_all_fees.feature [hunk 11]
```diff
@@ -5,14 +5,15 @@ Feature: Seeing order's payment amount including all fees
...
 
     Background:
-        Given the store operates on a single channel in "France"
-        And there is a zone "EU" containing all members of the European Union
-        And default tax zone is "EU"
-        And the store has "EU VAT" tax rate of 23% for "Clothes" within "EU" zone
-        And the store has a product "Angel T-Shirt" priced at "€39.00"
+        Given the store operates on a single channel in "United States"
+        And there is zone "NA" containing all members of the North America
+        And there is a zone "The Rest of the World" containing all other countries
```
Comment (michalmarcinkowski) [inline features/order/managing_orders/seeing_orders_payment_amount_including_all_fees.feature]: Irrelevant

File: features/shipping/managing_shipping_methods/adding_shipping_method.feature [hunk 12]
```diff
@@ -6,15 +6,15 @@ Feature: Adding a new shipping method
 
     Background:
         Given the store is available in "English (United States)"
-        And there is a zone "EU" containing all members of the European Union
+        And there is zone "NA" containing all members of the North America
```
Comment (michalmarcinkowski) [inline features/shipping/managing_shipping_methods/adding_shipping_method.feature]: there is **a "NA"** zone

File: src/Sylius/Component/Core/Test/Services/DefaultUnitedStatesChannelFactory.php [hunk 13]
```diff
@@ -157,7 +157,7 @@ public function create($code = null, $name = null)
         $currency = $this->provideCurrency();
         $locale = $this->provideLocale();
 
-        $channel = $this->createChannel($code ?: self::DEFAULT_CHANNEL_CODE, $name ?: 'France');
+        $channel = $this->createChannel($code ?: self::DEFAULT_CHANNEL_CODE, $name ?: self::DEFAULT_ZONE_NAME);
```
Comment (michalmarcinkowski) [inline src/Sylius/Component/Core/Test/Services/DefaultUnitedStatesChannelFactory.php]: We should use `DEFAULT_CHANNEL_NAME` here.

File: features/addressing/managing_zones/zone_validation.feature (hunk 1 above)
Comment (pjedrzejewski) [inline features/addressing/managing_zones/zone_validation.feature]: I believe it is correct.

**Comment by pjedrzejewski** [comment]
Thank you Jasiek, really nice work! :)

**Pull request merged**